### PR TITLE
fix: [IOBP-901] Missing header in payment methods

### PR DESCRIPTION
**Pull request by LeleDallas**
## Short description
This pull request includes a change to the `PaymentsMethodDetailsScreen` component to enhance the `getCardHeaderTitle` function. The update adds specific handling for BPAY and PayPal payment methods.

## List of changes proposed in this pull request
- Update `getCardHeaderTitle` function to handle missing edge cases

### io-dev-server
To test all card cases, update the `generateWalletData` function by adding this line: `generateUserWallet(3)`

### io-app

- Tap in _Portafoglio_
- BPAY or PayPal card
- Verify the correct card header title is displayed

### CODE REVIEW

File: ts/features/payments/details/screens/PaymentsMethodDetailsScreen.tsx [hunk 1]
```diff
@@ -90,14 +90,20 @@ const PaymentsMethodDetailsScreen = () => {
...
 
 const getCardHeaderTitle = (details?: UIWalletInfoDetails) => {
-  if (details?.lastFourDigits !== undefined) {
-    const capitalizedCardCircuit = capitalize(
-      details.brand?.toLowerCase() ?? ""
-    );
-    return `${capitalizedCardCircuit} ••${details.lastFourDigits}`;
+  switch (details?.type) {
+    case "BPAY":
+      return "BPAY";
```
Comment (thisisjp) [inline ts/features/payments/details/screens/PaymentsMethodDetailsScreen.tsx:95]:
```suggestion
      return "BANCOMAT Pay";
```

Comment (LeleDallas) [reply ts/features/payments/details/screens/PaymentsMethodDetailsScreen.tsx:95]: Thanks!😁

**Review by Hantex9 — APPROVED** [review]
LGTM!
